get-started-with-tidb-lightning.md

```diff
@@ -79,16 +86,14 @@ TiDB Lightning 的安装包位于 TiDB 离线工具包中。下载方式，请
     pd-addr = "172.16.31.3:2379"
```

Comment:
why the tidb host and the PD address is a strange IP address here ?  If it is a hands-on tutorial,  the TiDB is by default installed on 127.0.0.1 , right ?

Reply:
I'm not sure why the config file was set as such. Can you take a look at other parameters in the config and see if they are wrong as well?